frontend/src/App.js

```diff
@@ -1,18 +1,55 @@
 import { ThemeProvider } from '@mui/material'
 
 import './App.sass'
-import Navbar from './components/Navbar'
+import TaskCard from './components/TaskCard'
 import lambdeeTheme from './lambdeeTheme'
-import Sidebar from './components/Sidebar'
 import ErrorCounter from './components/ErrorCounter'
 
+const task = {
+  taskLabel: 'Fix login issue',
+  taskCategories: [['Frontend', '#FFFFFF', '#33E3FF'], ['Backend', '#000000', '#FF3333']],
```

Comment:
To zdecydowanie nie powinna być tablica, tylko jakiś obiekt. Jakbym to pisał w Ruby to stworzyłbym po prostu klasę osobną na to z trzema własnościami `label`, `textColor`, `backgroundColor` czy coś. Tutaj chyba wystarczy zwykły JSowy obiekt